actions/download-spctl/action.yaml

```diff
@@ -11,6 +11,9 @@ inputs:
     default: public
   gh_token:
     description: GH token for downloading private SPCTL
+  beta:
```

Comment:
Let's do this as part of "version" input. No need 2 variables that responsible for version.
```
  version:
    description: spctl version should be downloaded or latest or latest-beta
    default: latest
```